third_party/bigframes_vendored/ibis/expr/operations/analytic.py

```diff
@@ -2,22 +2,24 @@
 
 from __future__ import annotations
 
-from ibis.expr.operations.analytic import Analytic
+import ibis.expr.datatypes as dt
+import ibis.expr.operations.analytic as ibis_ops_analytic
+import ibis.expr.operations.core as ibis_ops_core
```

Comment:
I suggest to just simply use `import ibis.expr.operations as ops` since all op types are exposed there.